integration-tests/s3_cp.py

```diff
@@ -0,0 +1,12 @@
+import io
+import sys
+
+import smart_open
+
+s3_url, local_path = sys.argv[1:]
+with smart_open.smart_open(s3_url, 'rb') as fin, open(local_path, 'wb') as fout:
```

Comment:
description needed: I see copy s3 -> local disk, what you test here?

Reply:
This is an analog of the "aws s3 cp" command. I used it for debugging only, so I deleted it.